tests/PHPStan/Analyser/data/tagged-unions.php

```diff
@@ -169,7 +169,7 @@ public function doFoo3(array $a): void
 	public function doFoo4(array $a): void
 	{
 		// could be: array{a: int|string, b: string}
-		assertType('array{a: int, b: string}|array{a: string, b: string}', $a);
```

Comment:
Please delete the "could be" comments if the new asserted type is actually the one in the comment 😊